website/snippets/_new-sl-changes.md

```diff
@@ -0,0 +1,8 @@
+
+:::tip Introducing the new dbt Semantic Layer 🎉
+
+The dbt Semantic Layer has undergone a [significant revamp](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/), making it more efficient to define and query metrics. The new dbt Semantic Layer introduces new semantic components, improved governance, enhanced efficiency, data accessibility, and new Semantic Layer APIs. The new dbt Semantic Layer will lay the foundation for effective organization and interaction with data. The dbt Semantic Layer is available to all dbt Cloud plans (Developer, Team, and Enterprise) during [public beta](/docs/use-dbt-semantic-layer/dbt-sl#public-beta). 
```

Comment:
i think public beta should not be on this pag and be announced/detailed in the public beta release note. any reference to it can then link to that page. this way, it keeps this page clean and only maintains the relevant/current info